### DISCUSSION TITLE
How do you use WebTau?

### DISCUSSION THREAD

**Discussion by MykolaGolubyev** [testingisdocumenting/webtau#1177] [General]
Hello,
I am very curios to know how do you use Znai.
Your use cases will greatly help to prioritize existing and every growing backlog!

**Comment by LIttleAncientForestKami** [comment]
Znai or Webtau?

I'm not using Webtau yet, just looking around trying to see what it can help me with. I'm looking for something like ScanAPI for Python, but in Java.

**Reply by MykolaGolubyev** [reply]
ScanApi looks pretty cool. I believe similar result can be achieved with WebTau for testing. WebTau can also capture test artifacts for documentation purposes. If you chose to use Znai it can consume those artifacts with ease. But any other documentation system should be able to consume them as well as they are in json format. 

Let me know if you need any help!
